lm75bd/lm75bd.c

```diff
@@ -28,7 +28,43 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  error_code_t errCodeSend;
+  error_code_t errCodeReceive;
+  if (temp == NULL) {
+    return ERR_CODE_INVALID_ARG;
+  } 
+  uint8_t tempWriteByte = 0x00; // 0x00 because that's what the notion document specified for the i2cSendTo() function
+ 
+  errCodeSend = i2cSendTo(devAddr, &tempWriteByte, 1);
+  if (errCodeSend != ERR_CODE_SUCCESS) {
+    return ERR_CODE_INVALID_ARG;
+  }
+  uint8_t tempData[2] = {0};
+  errCodeReceive = i2cReceiveFrom(devAddr, tempData, 2);
+  if (errCodeReceive != ERR_CODE_SUCCESS) {
+    return ERR_CODE_INVALID_ARG;
```

Comment:
return the error code variable instead of hard coding the return since the error code could've been anything